x-pack/test/api_integration/services/security_solution_api.gen.ts

```diff
@@ -294,6 +295,20 @@ If asset criticality records already exist for the specified entities, those rec
         .set(ELASTIC_HTTP_VERSION_HEADER, '2023-10-31')
         .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana');
     },
+    /**
+     * Configuring the Risk Engine Saved Object
+     */
+    configureRiskEngineSavedObject(
+      props: ConfigureRiskEngineSavedObjectProps,
+      kibanaSpace: string = 'default'
+    ) {
+      return supertest
+        .patch(routeWithNamespace('/api/risk_score/engine/saved_object/configure', kibanaSpace))
```

Comment:
General open question about this path: I noticed that wrapping this in an interface and adding it to the consts file is a nice approach. However, I see the same path being hardcoded here and in the YAML file. Would it make sense to extract the path itself into a constant to make updates and maintenance easier?

Reply:
These .gen.ts files are generated automatically for us by the OpenAPI doc generator 👍